ckanext/s3filestore/uploader.py

```diff
@@ -371,8 +371,8 @@ def update_data_dict(self, data_dict, url_field, file_field, clear_field):
                     pass
             data_dict[url_field] = self.filename
             self.upload_file = _get_underlying_file(self.upload_field_storage)
-            log.debug("ckanext.s3filestore.uploader: is allowed upload type: filename: %s, upload_file: %s, data_dict: %s",
-                      self.filename, self.upload_file, data_dict)
+            log.debug("ckanext.s3filestore.uploader: is allowed upload type: filename: %s, upload_file: %s",
```

Comment:
can you do a log.trace with data_dict?

Reply:
Possibly, but it's really not needed. This log statement is primarily about checking which upload types are allowed, we don't need the whole dict for that.